runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/catalog/CatalogHandler.java

```
@@ -114,4 +115,9 @@ default int decodePadding(
     {
         return 0;
     }
+
+    default String location()
+    {
+        return "";
```

This should throw `new UnsupportedOperationException("location")` instead.